Foundation/NSObjCRuntime.swift

```diff
@@ -266,22 +266,143 @@ public func NSStringFromClass(_ aClass: AnyClass) -> String {
     neither stable nor human-readable.
  */
 public func NSClassFromString(_ aClassName: String) -> AnyClass? {
-    let aClassNameWithPrefix : String
-    let components = aClassName._bridgeToObjectiveC().components(separatedBy: ".")
-    
-    switch components.count {
-    case 1:
-        guard !aClassName.hasPrefix("_Tt") else {
-            NSLog("*** NSClassFromString(\(aClassName)): cannot yet decode mangled class names")
+    switch aClassName {
```

Comment:
Since we have a default case can't we remove the specific cases where the String and class match, e.g. `NSAffineTransform`, that should reduce the number of entries